README.md

```diff
@@ -27,9 +27,9 @@ This repository provides a modern, well-structured template for developing WordP
 
 2. **Update plugin information**
    - Edit `plugin.php` and replace the placeholder values:
-     - `bcgov-plugin-template` with your plugin name
-     - `{plugin-name}` with your plugin slug
-     - `{PluginName}` with your plugin's namespace
+     - `{bcgov-plugin-template}` with your plugin name
+     - `bcgov-plugin-template` with your plugin slug
+     - `{bcgov-plugin-template}` with your plugin's namespace
```

Comment:
Needs work: This is not right either. There were different cases (`Plugin Name` for title case like when the plugin is shown on WordPress plugins page, `PluginName` for Pascal case like class names where our linting rules require a certain case, `plugin-name` for kebab case like slugs and the package.json name where the value must be lowercase) before that need to be preserved.
Probably easiest to revert your changes since the last review and only replace exactly the text `{plugin-name}` with exactly `bcgov-plugin-template`.